app/pages/help/help-page.js

```diff
@@ -1,8 +1,27 @@
-
-exports.onNavigatingTo = function(args) {
-  if (args.isBackNavigation) {
-    return;
+  var frameModule = require("ui/frame");
+  var HelpViewModel = require("./help-view-model");
+  
+  var helpViewModel = new HelpViewModel();
+  
+  function pageLoaded(args) {
+    var page = args.object;
+    page.bindingContext = helpViewModel;
   }
 
-  // do something
-};
+  
+  function showItems(args) {
+    const container = args.view;
+    const instructions = container.getChildAt(1);
+    const flexLayout = container.getChildAt(0);
+  
+    const arrow = flexLayout.getChildAt(1);
+  
+    instructions.visibility = instructions.bindingContext.collapsed ? 'visible' : 'collapse';
+    arrow.text = !instructions.bindingContext.collapsed ? '\uf107' : '\uf106';
+  
+    instructions.bindingContext.collapsed = !instructions.bindingContext.collapsed;
+  }
+  
+  exports.showItems = showItems;
```

Comment:
instead of exporting at the end of the file, you can do

`exports.showItems = function(args) { }` directly. It makes sense to export at the end of file, only in a large component.